app/src/main/java/org/simple/clinic/main/TheActivityComponent.kt

```diff
@@ -149,7 +152,10 @@ interface TheActivityComponent :
     NextAppointmentCardView.Injector,
     CriticalAppUpdateDialog.Injector,
     EnterDrugStockScreen.Injector,
-    OverdueSearchScreen.Injector {
+    OverdueSearchScreen.Injector,
+    BloodSugarEntrySheet.Injector,
+    ConfirmRemoveBloodSugarDialogInjector,
+    BloodSugarUnitSelectionDialogInjector {
```

Comment:
Why have we added `BloodSugarUnitSelectionDialogInjector` in this PR?

Reply:
Previously it was part of the `BloodSugarEntryComponent`. So, we have to move that as well